README.md

```diff
@@ -1 +1,5 @@
-# project-tabular-data
\ No newline at end of file
+# Project-tabular-data
```

Comment:
Keeping the name consistent with Repository name would be nice. Wdyt?

Reply:
It's already the same right? Or are you referring to the case?
In that case, sentence-case is much more readable

Reply:
Yeah, I was talking about the case.